analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/LocksReleasedAllPaths.Mutex.cs

```diff
@@ -279,4 +278,58 @@ public void CompliantComplex(string mutexName, bool shouldAcquire)
             }
         }
     }
+
+    public void MutextAquireByConstructor_SimpleAssignment_LiteralArgument()
+    {
+        var m = new Mutex(true, "bar", out var mutextCreated); // Noncompliant
```

Comment:
This looks almost the same as line 13. Line 13 is still FN because of #5370 ?

Reply:
Yes, it's almost the same case. The problem seems to be in the engine and not in the rule code.